sources_test.go

```diff
@@ -111,3 +113,22 @@ func TestArgs(t *testing.T) {
 	}
 
 }
+
+func TestGet(t *testing.T) {
+	defer gock.Off()
```

Comment:
Would you mind very much using plain old `httptest` for this instead? I'm rather pleased with the fact that `script` is currently a zero-import library.

Reply:
For example: https://github.com/bitfield/checkly/blob/master/checkly_test.go#L23

Reply:
So you would rather do a full integration test here instead of a test with mocks (*avoiding any 3rd-party dependencies*)?

Reply:
No, an integration test would actually call out to the network. I'm suggesting using a local HTTP server, as you do here; just not using the `gock` library.